### ISSUE TITLE
Issue with culture code

### ISSUE THREAD

**Issue by SteJW** [buckaroo-it/BuckarooSDK_PHP#132]
{
    "message": "Buckaroo SDKExeption: Status code: 400 Message: Culture header could not be parsed. The provided value is not a known culture.",
    "exception": "Buckaroo\\Exceptions\\BuckarooException",
}

Even when I add culture code to the call, it doesn't work. In 1.6.x there is no such error.

**Comment by ShuCh3n** [comment]
Could you show me your code?

**Comment by SteJW** [comment]
We used to not send a culture Code and it's possible that we don't use send the server variable either. Code looks like:

```
                      $buckaroo = new BuckarooClient($config['BPE_WEBSITE_KEY'], $config['BPE_SECRET_KEY'], $config['BPE_MODE']);
                      $data = (new TransactionRequest())->setPayload(new PayPayload([
                          'currency' => $config['BPE_CURRENCY_CODE'],
                          'amountDebit' => (int)$price,
                          'invoice' => $paymant_id,
                          'description' => $event->submission->form()->title.' '.$paymant_id,
                          'continueOnIncomplete' => 1,
                          'servicesSelectableByClient' => $config['BPE_SERVICES_SELECTABLE_BY_CLIENT'],
                          'returnURL' => $config['BPE_RETURN_URL'],
                          'pushURL' => $config['BPE_PUSH_URL'],
                      ]));
```

**Comment by ShuCh3n** [comment]
Try this

```php
$buckaroo = new BuckarooClient($config['BPE_WEBSITE_KEY'], $config['BPE_SECRET_KEY'], $config['BPE_MODE']);
$response = $this->buckaroo->method(null)->pay([
            'currency' => $config['BPE_CURRENCY_CODE'],
            'amountDebit' => (int)$price,
            'invoice' => $paymant_id,
            'description' => $event->submission->form()->title.' '.$paymant_id,
            'servicesSelectableByClient' => $config['BPE_SERVICES_SELECTABLE_BY_CLIENT'],
            'continueOnIncomplete' => '1',
            'returnURL' => $config['BPE_RETURN_URL'],
            'pushURL' => $config['BPE_PUSH_URL']
        ]);

```

**Comment by dvdheiden** [comment]
As it might help others (or my future self), you could provide the culture to the config since 1.7.0:

```php
$config = new DefaultConfig(
    websiteKey: $config['BPE_WEBSITE_KEY'],
    secretKey: $config['BPE_SECRET_KEY'],
    mode: $config['BPE_MODE'],
    culture: $config['BPE_CULTURE'],
);

$buckaroo = new BuckarooClient($config);
```

**Comment by ShuCh3n** [comment]
We fixed this in #135
